c2siorg/Codelabz
# Social Media Icon Buttons are not Working #88 Fixed

Social Media Icon Buttons are not Working properly in profile page. while clicking on them they are not redirecting to the particular path is Fixed.

## Description

I changed the Social Media Icons file with adding a working functionality to it and I get the url data from the current user.

## Related Issue

Social Media Icon Buttons are not Working resolve #88 

### steps to reproduce

- Set-up the project in your localhost.
- Now run and go to the profile page.
- You find icons of facebook, github,linkedin etc.
- Click on them they are not redirecting before clicking make sure that you have provided the user profile name in the user settings path for that social media.

https://github.com/c2siorg/Codelabz/assets/106808387/9c0ba4ec-a922-4dfd-874c-fa06597fc3bc

#### Link of the Issue

[https://github.com/c2siorg/Codelabz/issues/88](url)


## Motivation and Context

Social Media Icons Buttons are used to redirect the user to the particular social media platform. Now a days Social media plays a crucial role. If the buttons not redirecting to that particular platform. It will be a big drawback. Now I Fixed This issue. Now User Can redirect to the particular platform.

## How Has This Been Tested?

- Set-up the project in your localhost.
- Now run and go to the profile page.
- You find icons of facebook, github,linkedin etc.
- Click on them they must be redirecting before clicking make sure that you have provided the user profile name in the user settings path for that social media.

## Screenshots or GIF (In case of UI changes):

https://github.com/c2siorg/Codelabz/assets/106808387/c25d13c1-e46b-4813-a65d-f81f4c905e91

## Types of changes


## Checklist:


## Review Request

@Maahi10001  @ABHISHEK-PANDEY2  @Shiva-Nanda  @c2sibot  Please Review My PR. Approve My PR
